app/services/users/merge_accounts_service.rb

```diff
@@ -0,0 +1,188 @@
+module Users
+  class MergeAccountsService
+    def initialize(old_user, new_user, student_profile_ids: [])
```

Comment:
Because the _type_ of the first two arguments are identical, how about we switch to mandatory keyword arguments here?

```suggestion
    def initialize(old_user:, new_user:, student_profile_ids: [])
```